pkg/ottl/go.mod

```diff
@@ -4,6 +4,8 @@ go 1.22.0
 
 require (
 	github.com/alecthomas/participle/v2 v2.1.1
+	github.com/antchfx/xmlquery v1.4.1
+	github.com/antchfx/xpath v1.3.1
```

Comment:
Have you used these dependencies before?

Reply:
I haven't other than the last few days. It appears to be at least somewhat widely used. My experience with it has been positive so far.